test/suite/test_tiered19.py

```diff
@@ -252,29 +252,32 @@ def test_ss_azure_file_system(self):
         self.assertEqual(ss.ss_flush_finish(session, azure_fs, 'non_existing_file', 'non_existing_file', None), 0)
 
         # Test that the no new objects exist after failed flush.
-        self.assertEquals(azure_fs.fs_directory_list(session, '', prefix_1), [prefix_1 + 'foobar'])
+        self.assertEquals(azure_fs.fs_directory_list(session, None, None), [prefix_1 + 'foobar'])
 
         err_msg = '/Exception: Operation not supported/'
 
         # Test that POSIX Remove and Rename are not supported.
         self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
             lambda: azure_fs.fs_remove(session, 'foobar', 0), err_msg)
-        self.assertEquals(azure_fs.fs_directory_list(session, '', prefix_1), [prefix_1 + 'foobar'])
+        self.assertEquals(azure_fs.fs_directory_list(session, None, None), [prefix_1 + 'foobar'])
 
         self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
             lambda: azure_fs.fs_rename(session, 'foobar', 'foobar2', 0), err_msg)
-        self.assertEquals(azure_fs.fs_directory_list(session, '', prefix_1), [prefix_1 + 'foobar'])
+        self.assertEquals(azure_fs.fs_directory_list(session, None, None), [prefix_1 + 'foobar'])
 
         # Flush second valid file into Azure.
         self.assertEqual(ss.ss_flush(session, azure_fs, 'foobar', 'foobar2', None), 0)
         # Check that second file exists in Azure.
         self.assertEqual(ss.ss_flush_finish(session, azure_fs, 'foobar', 'foobar2', None), 0)
 
         # Directory list should show 2 objects in Azure.
-        self.assertEquals(azure_fs.fs_directory_list(session, '', prefix_1), [prefix_1 + 'foobar', prefix_1 + 'foobar2'])
+        self.assertEquals(azure_fs.fs_directory_list(session, None, None), [prefix_1 + 'foobar', prefix_1 + 'foobar2'])
 
         # Directory list single should show 1 object.
-        self.assertEquals(azure_fs.fs_directory_list_single(session, '', prefix_1), [prefix_1 + 'foobar'])
+        self.assertEquals(azure_fs.fs_directory_list_single(session, None, None), [prefix_1 + 'foobar'])
+
+        # File system size on foobar should result in the length of foobar.
```

Comment:
It might be better to say that "Verify that file system size returns the size in bytes of the 'foobar" object." It's a bit uncommon to use the term "length" when referring to a file size.

Reply:
resolved https://github.com/wiredtiger/wiredtiger/pull/8804/commits/66d81186597758a131f6e70dc2835e8690a63718